src/routes/+page.svelte

```diff
@@ -40,10 +40,11 @@
 	.wrap {
 		margin-inline: auto;
 		width: min(87%, 100rem);
-		margin-top: 7rem;
 	}
 	.wrappezoid {
-		height: calc(100vh - 120px);
+		height: 100%;
+		padding-top: 2.5rem;
+		min-height: calc(100vh/10*7);
```

Comment:
what's this needed for?

Reply:
New calculation makes the home page look more visually balanced regardless of screen size.

Original:
![image](https://github.com/ReVanced/revanced-website/assets/56329333/990a52c7-f438-4001-9769-3ef2fcc0215d)

New:
![image](https://github.com/ReVanced/revanced-website/assets/56329333/95578c55-fbac-4429-a9aa-3dce5ec5e8ea)

Reply:
It seems to move the preview image a bit too high on my device, it's almost touching the navbar

Before:
![image](https://github.com/ReVanced/revanced-website/assets/79272171/aa259c35-7a79-4ace-b59c-3754be7b2a83)

After:
![image](https://github.com/ReVanced/revanced-website/assets/79272171/c8544967-882a-48cc-b085-b62d1e33c883)

Reply:
@404-Program-not-found Is this PR still active?

Reply:
> @404-Program-not-found Is this PR still active?

yup, I'll look into that. Sorry, life just got a bit busy recently